sphere-common/src/test/java/io/sphere/sdk/queries/PagedQueryResultTest.java

```diff
@@ -10,7 +10,7 @@
 
 public class PagedQueryResultTest {
 
-    public static final int TOTAL = 100;
+    public static final Long TOTAL = 100L;
     public static final int PAGE_SIZE = 25;
```

Comment:
Maybe better to have `Long` and pass `PAGE_SIZE.intValue()` to `listOfSize`. It feels cleaner than casting :)